cleanlab/rank.py

```diff
@@ -427,6 +429,133 @@ def get_label_quality_ensemble_scores(
+
+    The score is between 0 and 1; lower scores
+    indicate labels less likely to be correct. For example:
+
+    1 - clean label (the given label is likely correct).
+    0 - dirty label (the given label is unlikely correct).
+
+    Parameters
+    ----------
+    labels_multiannotator : pd.DataFrame
+        2D pandas DataFrame of multiple given labels for each example with shape (N, M),
+        where N is the number of examples and M is the number of annotators.
+        labels_multiannotator[n][m] = given label for n-th example by m-th annotator.
+        For a dataset with K classes, the given labels must be an integer in 0, 1, ..., K-1 or
+        np.nan if not annotated. Column names should correspond to the annotators' ID.
+
+    pred_probs : np.array
+        An array of shape ``(N, K)`` of model-predicted probabilities,
+        ``P(label=k|x)``. Each row of this matrix corresponds
+        to an example `x` and contains the model-predicted probabilities that
+        `x` belongs to each possible class, for each of the K classes. The
+        columns must be ordered such that these probabilities correspond to
+        class 0, 1, ..., K-1.
+
+        **Caution**: `pred_probs` from your model must be out-of-sample!
+        You should never provide predictions on the same examples used to train the model,
+        as these will be overfit and unsuitable for finding label-errors.
+        To obtain out-of-sample predicted probabilities for every datapoint in your dataset, you can use :ref:`cross-validation <pred_probs_cross_val>`.
+        Alternatively it is ok if your model was trained on a separate dataset and you are only evaluating
+        data that was previously held-out.
+
+    consensus_method : str or List[str]
+        Describing which algorithm to reach the consensus label.
+        If a List is passed, then the 0th element of List is the method used to produce
+        consensus_label, quality_of_consensus, annotator_agreement (may or may not be affected by this argument).
+        The 1st, 2nd, 3rd, etc. elements of this List are output as extra columns in the returned DF with names formatted as:
+        consensus_label_SUFFIX, quality_of_consensus_SUFFIX
+        where SUFFIX = the str element of this list.
+
+    kwargs : Same additional arguments as for get_label_quality_scores().
+
+    Returns
+    -------
+    df : pandas DataFrame in which each row corresponds to one example, with columns:
+                quality_annotator_1, quality_annotator_2, ..., quality_annotator_M, annotator_agreement, consensus_label, quality_of_consensus
+            (Here _1, _2, ..., _M suffixes may be replaced column names used to describe annotators in `labels` input)
+    """
+
+    # Compute the overall label health score for each annotator (part of annotator_stats)
+    overall_label_health_score_df = labels_multiannotator.apply(
+        overall_label_health_score, args=[pred_probs], verbose=False
```

Comment:
for these `apply()` calls, should `args = (pred_probs,)` as tuple instead or does not it not matter?